### ISSUE TITLE
Using Swift Package Manger fails to build a macOS target

### ISSUE THREAD

**Issue by adamtharani** [mixpanel/mixpanel-iphone#940]
**Integration Method:** Swift Package Manager
**Xcode Version:** 12.5
**Library Version:** 4.0.0 beta
**Platform:** iOS & macOS
**Language:** Objective-C
**Description:** 
```

Showing Recent Errors Only
SourcePackages/checkouts/mixpanel-iphone/Sources/AutomaticEvents.h:9:9: UIKit is not available when building for macOS. Consider using `#if !TARGET_OS_OSX` to conditionally import this framework, first importing TargetConditionals.h if necessary.

```
**Expected Behavior:** 

Build doesn't fail

**Comment by adamtharani** [comment]
Any update on this?

**Comment by zihejia** [comment]
hi @adamtharani , sorry for the late response. We will fix this.

**Comment by zihejia** [comment]
hi @adamtharani , it's been fixed in [4.1.3](https://github.com/mixpanel/mixpanel-iphone/releases/tag/v4.1.3). I'm closing it now. Please feel free to reopen it if the problem still exists.
